File: shortcut.py
from os import environ, getcwd as getworkdir
from win32com.client import Dispatch

workdir = getworkdir()
shell = Dispatch("WScript.Shell")

shortcut = shell.CreateShortCut(environ["HOMEPATH"] + r"\Desktop\TimeTable.lnk")
shortcut.Targetpath = workdir + r"\bin\javaw.exe"
shortcut.IconLocation = workdir + r"\icon.ico"
shortcut.Arguments = fr"-jar {workdir}\TimeTable.jar"
shortcut.WorkingDirectory = workdir
shortcut.save()

File: Shortcut.cs
using System;
using System.IO;
using IWshRuntimeLibrary;

namespace Shortcut {
    public class Shortcut {
        public static void Main(string[] args) {
            var workdir = Directory.GetCurrentDirectory();
            var desktop = Environment.GetFolderPath(Environment.SpecialFolder.Desktop);
            var shortcut = (IWshShortcut)new WshShell().CreateShortcut(desktop + @"\TimeTable.lnk");
            shortcut.TargetPath = workdir + @"\bin\javaw.exe";
            shortcut.IconLocation = workdir + @"\icon.ico";
            shortcut.Arguments = $"-jar \"{workdir}\\TimeTable.jar\"";
            shortcut.WorkingDirectory = workdir;
            shortcut.Save();
        }
    }
}